packages/prelude/src/index.js.flow

```diff
@@ -30,6 +30,8 @@ declare export function isArrayLike(x: any): boolean
 
 declare export function id<A>(A): A
 
+declare export function pipe <A, B, C>(f: (a: A) => B, g: (b: B) => C): (x: A) => C
```

Comment:
You can omit the parameter names inside the function parameters.  See `compose` below.